packages/core/src/actions/getTaskQueuePaused.ts

```diff
@@ -0,0 +1,20 @@
+import { getWalletId } from './getWalletId.js'
+
+import { getRelayerWithAuth } from '../utils/http.js'
+
+import { GET_TASK_QUEUE_PAUSED_ROUTE } from '../constants.js'
+import type { Config } from '../createConfig.js'
+
+export type GetTaskQueuePausedReturnType = Promise<boolean>
+
+export async function getTaskQueuePaused(
+  config: Config,
+): GetTaskQueuePausedReturnType {
```

Comment:
`GetTaskQueuePausedReturnType` should be boolean, because when the function is awaited the return type is the boolean. the return signature should then be changed to `Promise<GetTaskQueuePausedReturnType>` to reflect this. I've run into type issues regarding this before which is why I've been slowly migrating functions over to this type pattern.